config/v1/types_infrastructure.go

```diff
@@ -1769,9 +1769,7 @@ type NutanixFailureDomain struct {
 	// obtained from the Prism Central console or using the prism_central API.
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:MinItems=1
-	// +kubebuilder:validation:MaxItems=1
```

Comment:
How many subnets can we support? Not unlimited, we have to pick a sensible maximum

Reply:
changed.